RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 -----
 * [*] [internal] Refactored fetching posts in the Reader tab, including post related operations (i.e. like/unlike, save for latter, etc.) [#20197]
 * [**] Reader: Add a button in the post menu to block an author and stop seeing their posts. [#20193]
+* [**] [Jetpack-only] Help: Display the Jetpack app FAQ card on Help screen when switching from the WordPress app to the Jetpack app is complete. [#20232]
```

Comment:
@guarani, using "switching" in the release notes based on p1677768648136459-slack-C0180B5PRJ4

Made the same comment on [Android](https://github.com/wordpress-mobile/WordPress-Android/pull/18032).

Reply:
Ok, sounds good @staskus!